fuzzers/aflcc/fuzzer.py

```diff
@@ -272,6 +272,7 @@ def run_fuzzer(input_corpus,
         # performs.
         '2147483647'
     ]
+    os.environ['AFL_SHUFFLE_QUEUE'] = '1'
```

Comment:
This wont be needed since it calls into afl's prepare_fuzz_environment.

Reply:
ok